### PR TITLE
fix: replace deprecated header for humble

### PR DESCRIPTION
**Pull request by soblin**
Maybe this package is not recently used, but I'd like to submit this change just in case.

Signed-off-by: Mamoru Sobue <mamoru.sobue@tier4.jp>

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -10,6 +10,12 @@ include_directories(
   ${PCL_COMMON_INCLUDE_DIRS}
 )
 
+if(${tf2_sensor_msgs_VERSION} VERSION_LESS 0.18.0)
+  target_compile_definitions(${PROJECT_NAME} PUBLIC
```
Comment (wep21) [inline CMakeLists.txt]: You need to set target_compile_definitions after creating the target.

**Review by soblin — COMMENTED** [review]
~add target_compile_definition~

**Comment by soblin** [comment]
Thank you for your support. I could fix the error !

File: src/laserscan_to_pointcloud_node.cpp [hunk 2]
```diff
@@ -49,7 +49,11 @@
 #include <utility>
 
 #include "sensor_msgs/point_cloud2_iterator.hpp"
-#include "tf2_sensor_msgs/tf2_sensor_msgs.h"
+#ifdef USE_TF2_SENSOR_MSGS_DEPRECATED_HEADER
+#include <tf2_sensor_msgs/tf2_sensor_msgs.h>
```
Comment (wep21) [inline src/laserscan_to_pointcloud_node.cpp]: @soblin [NITS]: I think it is better to keep double quotes unless you have any specific reason to change them to brackets.

**Review by wep21 — APPROVED** [review]
LGTM. Thank you.

**Pull request merged**